packages/ckeditor5-list/src/utils.js

```diff
@@ -365,6 +375,40 @@ export function getSiblingNodes( position, direction ) {
 	return items;
 }
 
+/**
+ * Returns an array with all `listItem` elements that represents the currently selected list.
+ *
+ * It returns all the items even if only part of the list is selected.
+ * If no list is selected, returns an empty array.
+ * The order of elements is not specified.
+ *
+ * @param {module:engine/model/model~Model} model
+ * @returns {Array.<module:engine/model/element~Element>}
+ */
+export function getListItemsOfSelectedList( model ) {
```

Comment:
I suppose this should be `@Protected`. There's no need to make it part of the public API.